doc/prelimsLinux.rst

```diff
@@ -0,0 +1,222 @@
+Preliminary Steps for Linux Distributions
+=========================================
+
+1) Install Python 3.6+
+----------------------
+Check your current python version(s) by entering the following commands into the command line: 
+
+.. code-block:: shell
+
+   python --version
+   python3 --version
+   python3.6 --version
+
+If Python 3.6+ is not present, follow these instructions depending on your operating system:
+
+Running a **Linux** distribution, Python must be compiled. Download and compile the most recent Python release with the following commands (find the link to the correct .tar file `here <https://www.python.org/downloads/>`__):
```

Comment:
I agree in that I am not sure this is necessary. I decided to include it as I was working through installations just to be verbose. Could most likely be replaced with a link to python.org.